<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planterra</string>
  </property>
  <property name="windowOpacity">
   <double>8.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget_Main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_MainMenu">
     <widget class="QLabel" name="lable_AppName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Planterra</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>160</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_MainMenuButtons">
       <item>
        <widget class="QPushButton" name="pushButton_NewField">
         <property name="text">
          <string>New Field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_LoadField">
         <property name="text">
          <string>Load Field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_Simulation">
     <widget class="QGraphicsView" name="graphicsView_Simulation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_SimulationInteractions">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>171</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_SelectedFieldProps">
       <widget class="QPushButton" name="pushButton_simMenu">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>510</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Open simulation menu</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea_itemProps">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>171</width>
          <height>391</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>169</width>
           <height>389</height>
          </rect>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>171</width>
            <height>441</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_itemProps"/>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_simAddPlant">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>510</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_simDeletePlant">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>510</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_simLight">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>550</y>
          <width>61</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_simHumidity">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>550</y>
          <width>61</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Humidity</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_simSaveName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_simTime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0:0:0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_simInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_SimulationOptions">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>171</width>
          <height>601</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_SimulationOptionsButtons">
         <item>
          <widget class="QPushButton" name="pushButton_simBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simLoad">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simReturn">
           <property name="text">
            <string>Return to Main Menu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simQuit">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
